k8s/apiextensions/apiextensions.go

```diff
@@ -18,18 +18,25 @@ var (
 	deleteForegroundPolicy = metav1.DeletePropagationForeground
 )
 
-// Ops is an interface to perform kubernetes related operations on the crd resources.
+// Ops is an interface to the apiextensions client wrapper.
 type Ops interface {
 	CRDOps
 
 	// SetConfig sets the config and resets the client.
 	SetConfig(config *rest.Config)
 }
 
+// Interface is an interface to perform kubernetes related operations on the crd resources.
+type Interface interface {
```

Comment:
This interface is not being used anywhere. You need to use it in Ops

Reply:
Updated.